src/Scripting/CSharpTest/ScriptTests.cs

```diff
@@ -33,6 +33,28 @@ public void TestCreateScript()
         }
 
         [Fact]
+        [WorkItem(17779, "https://github.com/dotnet/roslyn/issues/17779")]
+        public async Task TestScriptWithConstVar()
```

Comment:
Does this work?

```
class var {}
const var = null;
```

It should, right?


---
In reply to: [130477811](https://github.com/dotnet/roslyn/pull/21223#discussion_r130477811) [](ancestors = 130477811)